packages/fx-core/resource/package.nls.json

```diff
@@ -406,35 +406,35 @@
   "core.createProjectQuestion.officeXMLAddin.taskpane.detail": "Customize the Ribbon with a button and create a dashboard in the Task Pane",
   "core.createProjectQuestion.officeXMLAddin.manifestOnly.title": "Add-in Project With only Manifest File",
   "core.createProjectQuestion.officeXMLAddin.manifestOnly.detail": "Create an add-in project that includes only the manifest file",
-  "core.aiAssistantBotOption.label": "AI Assistant Bot",
-  "core.aiAssistantBotOption.detail": "A custom AI assistant bot in Teams using Teams AI library and OpenAI Assistants API",
+  "core.aiAssistantBotOption.label": "AI Agent Bot",
```

Comment:
This might be an OpenAI specific term, "AI Assistant". Can check with dev or PM Junjie to confirm.

Reply:
I checked with Junjie. We've changed 'AI Assistant' to 'AI Agent'. Updated the title case. Thanks.